pkg/kubelet/kubelet_server_journal_linux.go

```diff
@@ -0,0 +1,73 @@
+//go:build linux
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// getLoggingCmd returns the journalctl cmd and arguments for the given nodeLogQuery and boot. Note that
+// services are explicitly passed here to account for the heuristics
+func getLoggingCmd(n *nodeLogQuery, services []string) (string, []string, error) {
+	args := []string{
+		"--utc",
+		"--no-pager",
+		"--output=short-precise",
+	}
+	if n.SinceTime != nil {
+		args = append(args, fmt.Sprintf("--since=%s", strings.TrimSuffix(n.SinceTime.UTC().String(), " +0000 UTC")))
```

Comment:
calling the default `String()` impl and trimming that suffix is an odd way to format the time... can we use https://pkg.go.dev/time#Time.Format to get the actual format we want?